<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IDetailsOvlArea</class>
 <widget class="QDialog" name="IDetailsOvlArea">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelName">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelTainted">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The waypoint was imported to QMapShack and was changed. It does not show the original data anymore. Please see history for changes. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../../resources.qrc">:/icons/32x32/Tainted.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolLock">
       <property name="toolTip">
        <string>Toggle read only mode. You have to open the lock to edit the item.</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/icons/32x32/UnLock.png</normaloff>
         <normalon>:/icons/32x32/Lock.png</normalon>:/icons/32x32/UnLock.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="labelColor">
         <property name="text">
          <string>Color</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboColor">
         <property name="iconSize">
          <size>
           <width>64</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Border width</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBorderWidth"/>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Style</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboStyle">
         <property name="iconSize">
          <size>
           <width>64</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="checkOpacity">
       <property name="text">
        <string>Opacity</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextBrowser" name="textCmtDesc">
     <property name="openLinks">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Points</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTreeWidget" name="treeWidget">
         <column>
          <property name="text">
           <string notr="true">#</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Position</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Hist.</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="CHistoryListWidget" name="listHistory"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CHistoryListWidget</class>
   <extends>QListWidget</extends>
   <header>helpers/CHistoryListWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
